packages/terra-form-select/src/_Frame.module.scss

```diff
@@ -101,6 +101,14 @@
     cursor: default;
     opacity: 0.4;
     pointer-events: none;
+
+    .arrow-icon {
+      background: var(--terra-select-disabled-arrow-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi"><path d="M48 12L24 36 0 12h48z"/></svg>'));
+      background-position: center;
+      background-repeat: no-repeat;
+      background-size: contain;
+      display: inline-block; // IE 10
+    }
```

Comment:
You might consider flattening this similar to the invalid icon below. 

`.is-invalid .arrow-icon`

It probably doesn't make too big of a difference but it helps prevent nesting depth issues down the road and is a little easier to read.

Reply:
It's probably worth mentioning this Icon will inherit the `opacity` applied to `is-disabled`.